contents/tutorials/toolbar.md

```diff
@@ -1,217 +0,0 @@
----
```

Comment:
This tutorial is actually impossible to view anymore since the URL redirects to Toolbar page on the posts hub. We could rename and update it, but most of whats in here is now in the docs, so I dont think we need it anymore